docdir and htmldir corrected for Debian (without version number)
--- a/Makefile.am
+++ b/Makefile.am
@@ -32,7 +32,7 @@
 #EXTRA_DIST = LICENSE CHANGELOG autotools.sh test/test_pep.c test/Makefile doc/html
 EXTRA_DIST = LICENSE ChangeLog autotools.sh doc/Doxyfile $(example_DATA)
 
-docdir = $(datadir)/doc/$(PACKAGE_NAME)-$(PACKAGE_VERSION)
+docdir = $(datadir)/doc/$(PACKAGE_NAME)
 
 if ENABLE_LIBRARY
 doc_DATA = AUTHORS ChangeLog LICENSE README
@@ -41,7 +41,7 @@
 
 # Generate doxygen doc
 doc: html
-htmldir = $(docdir)/api
+htmldir = $(docdir)/html
 if ENABLE_DEVEL
 doxygen: doc/Doxyfile
 	$(DOXYGEN) $(srcdir)/doc/Doxyfile
